luigi/scheduler.py

```diff
@@ -90,6 +90,8 @@ class scheduler(Config):
                                             config_path=dict(section='scheduler', name='disable-window-seconds'))
     disable_failures = parameter.IntParameter(default=None,
                                               config_path=dict(section='scheduler', name='disable-num-failures'))
+    disable_hard_timeout = parameter.TimeDeltaParameter(default=None,
+                                              config_path=dict(section='scheduler', name='disable-hard-timeout'))
```

Comment:
It should be automatic. Notice however it's spaces not underscore.

I'm not sure if I ever fixed it so that it tries both. Maybe I did (in which case the `config_path` argument does not matter)